<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Titlebar</class>
 <widget class="QWidget" name="Titlebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>35</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>apologizebao</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="iconLabel">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/icon/default.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="modeBtn">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/read.svg</normaloff>
       <normalon>:/icon/editor.svg</normalon>
       <disabledoff>:/icon/read.svg</disabledoff>:/icon/read.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="NLabel" name="settingLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>设置</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel:hover{
	image: url(:/icon/setting_focuse.svg);
}
QLabel:!hover{
	image: url(:/icon/setting.svg);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="NLabel" name="minLabel">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>最小化</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel:hover{
	image: url(:/icon/min_focuse.svg);
}
QLabel:!hover{
	image: url(:/icon/min.svg);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="NLabel" name="maxLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>最大化</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel:hover{
	image: url(:/icon/max_focuse.svg);
}
QLabel:!hover{
	image: url(:/icon/max.svg);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NLabel" name="closeLabel">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="toolTip">
      <string>关闭</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel:hover{
	image: url(:/icon/close_focuse.svg);
}
QLabel:!hover{
	image: url(:/icon/close.svg);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLabel</class>
   <extends>QLabel</extends>
   <header location="global">nlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
